### DISCUSSION TITLE
Charged for Copilot as a student

### DISCUSSION THREAD

**Discussion by FrancoMuniz** [community/community#30396] [Copilot Conversations]
Hello,

I've had GitHub Pro for several months as a student through the Student Pack. I also activated Copilot as a student when it came out of the free beta.

I activated my Student Pack on Nov 2020, if that's relevant.

A few hours ago, I was billed $10 for Copilot's monthly subscription.

1) I can't see any place to "toggle" the student mode, any ideas?
2) I cancelled my Copilot subscription to see if I could get to the student version that way, but now there's no way to re-activate it until the billing period ends. Should I wait?

Thanks.

**Comment by josyulakrishna** [comment]
Same issue for me.

**Reply by FrancoMuniz** [reply]
I contacted GitHub support and they issued a refund + fixed the issue

**Reply by josyulakrishna** [reply]
Thanks for the response. How did you contact the support? Is there any email?

**Reply by FrancoMuniz** [reply]
@josyulakrishna If you click "Account or billing issues" on this page, it should let you create a ticket: https://support.github.com/request

**Reply by josyulakrishna** [reply]
Thank you!

**Comment by FrancoMuniz** [comment]
If anyone had the same issue, simply contact support and they'll fix it
